metron-analytics/metron-maas-service/src/main/java/org/apache/metron/maas/submit/ModelSubmission.java

```diff
@@ -203,7 +203,7 @@ else if(mode.equalsIgnoreCase("LIST")) {
           String modelTitle = "Model " + kv.getKey().getName() + " @ " + kv.getKey().getVersion();
           System.out.println(modelTitle);
           for(ModelEndpoint endpoint : kv.getValue()){
-            System.out.println("\t" + endpoint.getContainerId() + " at " + endpoint.getUrl());
+            System.out.println(endpoint);
           }
```

Comment:
Actually, this the LIST operation is intended to output the list of endpoints returned.  What I think I *will* do is make some of the logging debug level because it's getting quite chatty in practice and redundant.